test/cli/cli_install_static_test.go

```diff
@@ -0,0 +1,57 @@
+package test
+
+import (
+	"flag"
+	"fmt"
+	"net/http"
+	"os"
+	"testing"
+
+	"github.com/linkerd/linkerd2/testutil"
+)
+
+var (
+	TestHelper *testutil.TestHelper
+)
+
+func TestMain(m *testing.M) {
+
+	// Read the flags and create a new test helper
+	exit := func(code int, msg string) {
+		fmt.Fprintln(os.Stderr, msg)
+		os.Exit(code)
+	}
+
+	linkerd := flag.String("linkerd", "", "path to the linkerd binary to test")
+	runTests := flag.Bool("cli-tests", false, "must be provided to run the cli tests")
+	flag.Parse()
+
+	if !*runTests {
+		exit(0, "cli tests not enabled: enable with -cli-tests")
+	}
+
+	if *linkerd == "" {
+		exit(1, "-linkerd flag is required")
+	}
+
+	TestHelper = testutil.NewGenericTestHelper(*linkerd, "", "l5d", "", "", "", "", "", "", false, false, *http.DefaultClient, testutil.KubernetesHelper{})
+	os.Exit(testutil.Run(m, TestHelper))
```

Comment:
`testutil.Run()` attempts to fetch the logs from the cluster whenever there's a failure. Since there's no cluster here, you can just do the old `os.Exit(m.Run())` instead.